posthog/clickhouse/log_entries.py

```diff
@@ -0,0 +1,68 @@
+from posthog.clickhouse.kafka_engine import KAFKA_COLUMNS, kafka_engine, ttl_period
+from posthog.clickhouse.table_engines import ReplacingMergeTree
+from posthog.kafka_client.topics import KAFKA_LOG_ENTRIES
+from posthog.settings import CLICKHOUSE_CLUSTER, CLICKHOUSE_DATABASE
+
+LOG_ENTRIES_TABLE = "log_entries"
+LOG_ENTRIES_TTL_WEEKS = 1
+
+LOG_ENTRIES_TABLE_BASE_SQL = """
+CREATE TABLE IF NOT EXISTS {table_name} ON CLUSTER '{cluster}'
+(
+    team_id UInt64,
+    log_source LowCardinality(String),
+    log_source_id VARCHAR,
+    instance_id VARCHAR,
+    timestamp DateTime64(6, 'UTC'),
+    level LowCardinality(String),
+    message VARCHAR
```

Comment:
Looks like I tend to use `String` and you prefer `VARCHAR` 

They're aliases IIRC so let's use one of them consistently so people don't have to wonder if they're the same in future